### DISCUSSION TITLE
2FA suddenly stopped working

### DISCUSSION THREAD

**Discussion by UP-MDK** [community/community#134729] [New to GitHub] [answered]
Hello,

i am using the microsoft authenticator app on my Samsung Galaxy S7. I have 2FA set up in Github along with the authenticator app. 2FA was working fine, the whole time. Suddenly it stopped working for no apparent reason.

So far, i checked the following articles and the possible solutions in it:

- https://github.com/google/google-authenticator-android/issues/56
- https://stackoverflow.com/questions/66856989/totp-2fa-codes-for-login-to-github-are-not-working-anymore-since-daylight-saving

I checked the time on my phone and since it is mid-summer, there can't be any issues with transitioning from winter to summer time. Also all other apps are working fine with the microsoft authenticator. The problem seems to only occur with Github.

At the moment, i am using the recovery keys to sign in, which is propably not the recommended way.

Since i am relatively new to Github, are there any potential things that i am missing?

**Comment by FlutterSmith** [comment]
If 2FA codes are not working on GitHub, try these steps:

Sync Time: Ensure your phone’s time is set to automatically update.
Re-add Account: Remove and re-add GitHub in the Microsoft Authenticator app.
Update Apps: Check for updates for both the Authenticator app and your phone’s OS.
If the problem persists, contact GitHub support for further assistance.

**Reply by UP-MDK** [reply]
My phone is set to automatically update the time already.
I checked the authenticator app and the phone os for updates (they are up to date, as they update automatically).
I also tried other authenticator apps, but for some reason, my phone fails to communicate with Github entirely.

I would like to contact the Github support, but the support portal does not provide a possibility, other than AI.
Would you mind providing me a Link where i can describe the issue?

**Reply by FlutterSmith** [reply]
You can contact GitHub support directly by following these steps:

Visit GitHub Support: Go to [GitHub Support](https://github.com/contact).

Sign In: Make sure you are signed in to your GitHub account.

Submit a Request: Scroll down to the bottom and click on "Submit a request" to access the form where you can describe your issue in detail.

Fill Out the Form: Provide all necessary details about the problem, including the steps you've already taken.

This should help you get in touch with GitHub support for personalized assistance.

**Reply by UP-MDK** [reply]
Thank you. And sorry. Now that i used my recovery code to sign in, i can actually create a ticket.

**Reply by FlutterSmith** [reply]
You're welcome! Please remember to mark the question as answered. If you need any more help, feel free to ask!

**Reply by UP-MDK** [reply]
Okay, sorry again. The support portal is still not working. Upon creating a ticket, i can only chosse between different AI options. The AI is generally unable to process any input i make and only responds with "Sorry, I don't understand that input. Please try again."

I really don't understand

**Comment by davevad93** [comment, marked as answer]
Hi @UP-MDK ,

I think you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Sign in, verification, or other account questions` and explain your problem with 2FA.

**Reply by UP-MDK** [reply]
Very nice. Thank you! Finally a form to fill out :)

**Reply by davevad93** [reply]
You're welcome @UP-MDK !

**Comment by c0sminghinea** [comment]
Hi! I have the same issue for a while now. Did you manage to find a solution? Thanks in advance.

**Comment by formbi-kupiec** [comment]
Yeah, something is very wrong with the solution they themselves forced upon users. I have passkeys and TOTP setup in my KeePassXC and both methods stopped working. I tried Librewolf, Firefox and Chromium, it didn't work in any of them. I had to use a recovery code.

**Comment by UP-MDK** [comment]
I didn't find a solution. Neither did the support. I just switched 2FA to SMS for the time being.
This method is working reliably for me.
